lib_xua/src/core/endpoint0/xua_ep0_descriptors.h

```diff
@@ -18,6 +18,14 @@
 #include "xua_hid_descriptor.h"
 #include "xud.h"
 
+// Enable BOS descriptor only when DFU is enabled since the only capability we advertise is the MSOS desc with DFU interface enumerating as WinUSB.
+// Enumerating with 0 capabilities doesn't seem to be allowed
+#if XUA_DFU_EN
+    #define XUA_ENABLE_BOS_DESC (1)
```

Comment:
since this is an internal define only rename _XUA

Reply:
done